buildenv/jenkins/common/build

```diff
@@ -246,8 +246,10 @@ def checkoutRef (REF) {
 def build() {
     stage('Compile') {
         timestamps {
-            sh "bash ./configure --with-freemarker-jar='$FREEMARKER' --with-boot-jdk='$BOOT_JDK' $EXTRA_CONFIGURE_OPTIONS"
-            sh "make $EXTRA_MAKE_OPTIONS all"
+            withEnv(BUILD_ENV_VARS_LIST) {
+                sh "${BUILD_ENV_CMD} bash ./configure --with-freemarker-jar='$FREEMARKER' --with-boot-jdk='$BOOT_JDK' $EXTRA_CONFIGURE_OPTIONS"
+                sh "${BUILD_ENV_CMD} make $EXTRA_MAKE_OPTIONS all"
```

Comment:
If `BUILD_ENV_CMD` has side-effects, they will happen twice: I don't think that's appropriate.

Reply:
Fixed: combined `configure` and `make` in one command.

Reply:
@keithc-ca ready for review, thank you